package.json

```diff
@@ -4,7 +4,8 @@
   "main": "index.js",
   "scripts": {
     "start": "NODE_ENV=development node index.js",
-    "stats": "NODE_ENV=development node stats.js"
+    "stats": "NODE_ENV=development node stats.js",
+    "stats:file": "NODE_ENV=development node stats.js generateFile=true"
```

Comment:
It would be nice to have `node stats.js file`

Reply:
`file` being the name of the file that will be generated? Or simply replace `generateFile` by `file`?

Reply:
Simple replace it by file without =true :smile: